node/src/actors/chain_manager/handlers.rs

```diff
@@ -1336,4 +1437,143 @@ mod tests {
             ]
         );
     }
+    #[test]
+    fn test_split_blocks_batch() {
+        use BlockBatches::*;
+        let mut sync_target = SyncTarget {
+            block: Default::default(),
+            superblock: Default::default(),
+        };
+        let superblock_period = 10;
+
+        // TODO: remove all the calls to b(x)
+        let b = |x| x;
```

Comment:
As the comment says, we could remove these lines and remove all calls to b(x) :smile: